client/components/fields/editor/Location.tsx

```diff
@@ -18,6 +18,7 @@ export class EditorFieldLocation extends React.PureComponent<IEditorFieldProps>
                     label={this.props.label ?? gettext('Location')}
                     value={get(this.props.item, field, this.props.defaultValue)}
                     disableSearch={true}
+                    disableAddLocation={false}
```

Comment:
This component should allow this option to be controlled via an optional prop, and then in setting this prop to false in the `AdvancedSearch` component: https://github.com/superdesk/superdesk-planning/blob/develop/client/components/AdvancedSearch/index.tsx#L138